mino/minogrpc/tracing/tracers.go

```diff
@@ -0,0 +1,77 @@
+package tracing
```

Comment:
I love the elegance of this simple file.

However, I had to re-read it many times and found the naming very confusing.
I've added a ton of comments, feel free to ignore them altogether if you feel I'm wrong :)